.gitlab-ci.yml

```diff
@@ -30,6 +30,24 @@ deploy_to_docker_registries:
     branch: main
     strategy: depend
   variables:
-    IMG_SOURCES: ghcr.io/datadog/dd-trace-js/dd-lib-js-init:$CI_COMMIT_SHA
+    IMG_SOURCES: ghcr.io/datadog/dd-trace-js/dd-lib-js-init:$CI_COMMIT_TAG
```

Comment:
java uses the SHA here: https://github.com/DataDog/dd-trace-java/blob/master/.gitlab-ci.yml#L136 is it different for node?

Reply:
ah yeah, seems to line up with the tag that is published: https://github.com/DataDog/dd-trace-js/pull/2340/files#diff-57b402748df77503e37b3b4f47152b3cfb6502f3d2d00c79d4512a77df802333R55

Reply:
Yeah since we're publishing the tag explicitly.